exp-1-merit/kafka-shared/modules/consumer/provider.tf

```diff
@@ -0,0 +1,20 @@
+terraform {
+  required_providers {
+    kafka = {
+      source = "Mongey/kafka"
+    }
+  }
+}
+
+provider "kafka" {
```

Comment:
Do we need to specify the provider in here? Won't it be "inherited" from where the module is used?